integration-cli/docker_cli_build_test.go

```diff
@@ -1924,6 +1927,134 @@ func TestBuildForceRm(t *testing.T) {
 	logDone("build - ensure --force-rm doesn't leave containers behind")
 }
 
+func IsKilled(err error) bool {
```

Comment:
can you move this to utils.go so it can be more easily seen by others can then potentially reused?